vllm/engine/arg_utils.py

```diff
@@ -249,6 +265,20 @@ def add_cli_args(
                             action='store_true',
                             default=EngineArgs.disable_custom_all_reduce,
                             help='See ParallelConfig')
+        parser.add_argument('--num-tokenizer-actors',
```

Comment:
It seems a bad idea to specialize this API to actors, which are a Ray-only concept. For example, we may use threads or native Python multiprocessing in the future. If we add this to vLLM, we'll need to change this API name then.